src/chesster.js

```diff
@@ -118,6 +118,169 @@ function leagueDMResponse(patterns, responseName) {
 leagueResponse(['captain guidelines'], 'formatCaptainGuidelinesResponse');
 leagueDMResponse(['captains', 'captain list'], 'formatCaptainsResponse');
 
+/* 
+ * strip components
+ * Splits the text. Adds each word to an object using the keys provided. 
+ *
+ * Each value, component, is made to be lower case.
+ *
+ * If the number of components is fewer than the number of words in the text,
+ * the remaining words will be joined as the final argument.
+ */
+function stripComponents(text, componentKeys){
```

Comment:
I'm not a big fan of the name "stripComponents".  I think I would prefer the name 'tokenize'.